modules/luci-base/luasrc/dispatcher.lua

```diff
@@ -564,13 +564,13 @@ local function session_setup(user, pass)
 			values = { token = sys.uniqueid(16) }
 		})
 
-		io.stderr:write("luci: accepted login on /%s for %s from %s\n"
```

Comment:
Please replace with `nixio.syslog()` and get rid of the extra logic.

Reply:
Replaced with syslog call.